tests/modules/commandfilter.c

```diff
@@ -188,9 +189,22 @@ void CommandFilter_CommandFilter(RedisModuleCommandFilterCtx *filter)
             RedisModule_CreateString(NULL, log_command_name, sizeof(log_command_name)-1));
 }
 
-void CommandFilter_CommandPostFilter(RedisModuleCommandFilterCtx *filter) {
-    RedisModule_Log(NULL, "notice", "is_succeed:%d, is_dirty:%d", 
-        RedisModule_CommandFilterCmdIsSucceeded(filter), RedisModule_CommandFilterDataIsDirty(filter));
+void CommandFilter_CommandPostFilter(RedisModuleCommandFilterCtx *ctx) {
+    int argc = RedisModule_CommandFilterArgsCount(ctx);
+    if (argc <= 1) {
+        return;
+    }
+
+    int is_dirty = RedisModule_CommandFilterDataIsDirty(ctx);
+
+    RedisModuleString *key = RedisModule_CommandFilterArgGet(ctx, 1);
+    const char *key_str = RedisModule_StringPtrLen(key, NULL);
+
+    RedisModuleKey *k = RedisModule_OpenKey(module_ctx, key, REDISMODULE_READ | REDISMODULE_WRITE);
+    if (is_dirty && !memcmp(key_str, "@expireKey", 10)) {
```

Comment:
why memcmp and not strcmp? (seems like it could lead to an invalid read).
or better yet, use RedisModule_StringCompare